lib/Runtime/Library/ArgumentsObject.cpp

```diff
@@ -85,20 +85,17 @@ namespace Js
             return scriptContext->GetLibrary()->GetNull();
         }
 
-        Var args = walker->GetPermanentArguments();
-        if (args == NULL)
-        {
-            args = JavascriptOperators::LoadHeapArguments(
-                funcCaller,
-                paramCount - 1,
-                walker->GetJavascriptArgs(),
-                scriptContext->GetLibrary()->GetNull(),
-                scriptContext->GetLibrary()->GetNull(),
-                scriptContext,
-                /* formalsAreLetDecls */ false);
-
-            walker->SetPermanentArguments(args);
```

Comment:
It is removed from there is well, no?

---
In reply to: [67768600](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67768600) [](ancestors = 67768600,67767669)